build-support/bin/cherry_pick.sh

```diff
@@ -37,8 +37,16 @@ fi
 COMMIT=$(gh pr view "$PR_NUM" --json mergeCommit --jq '.mergeCommit.oid')
 TITLE=$(gh pr view "$PR_NUM" --json title --jq '.title')
 CATEGORY_LABEL=$(gh pr view "$PR_NUM" --json labels --jq '.labels.[] | select(.name|test("category:.")).name')
+if [[ -z $CATEGORY_LABEL ]]; then
+  # This happens occasionally, e.g., when cherrypicking the same PR to different branches.
+  # Unclear why, but this may help ferret it out.
+  echo "Couldn't detect category label on PR. What's the label? (E.g., category:bugfix)"
```

Comment:
If the user has to type this out, maybe enumerate all the options? Alternatively, should the user go back and fix the original PR?

Reply:
The original PR has a category. There is a weird case where this logic fails to detect it.

Reply:
It's not common, but I've seen it happen, so this is to help detect it. Once we lock it down I'd rather error outright in this case.